deployments/development-install/helmfile.yaml

```diff
@@ -0,0 +1,188 @@
+releases:
+  - name: ocis
+    chart: ../../charts/ocis
+    namespace: ocis
+    values:
+      - externalDomain: ocis.kube.owncloud.test
+      - ingress:
+          enabled: true
+          ingressClassName: nginx
+          annotations:
+            nginx.ingress.kubernetes.io/proxy-body-size: 1024m
+
+      - logging:
+          level: debug
+
+      - insecure:
+          oidcIdpInsecure: true
+          ocisHttpApiInsecure: true
+
+      - services:
+          idm:
+            persistence:
+              enabled: true
+              chownInitContainer: true
+              accessModes:
+                - ReadWriteOnce
```

Comment:
Can the access mode be lifted out to a variable or  maybe reference another part of the helmfile? And also that it should be mentioned that this access mode will only work on a single-node cluster.

Reply:
Hmm, when thinking about it it doesn't require a single-node cluster, but it should be mentioned that this access mode will not work if the deployments scale up above a single pod.

Reply:
This is true, also, afaik, ReadWriteMany is still relatively rare in most setups.

Reply:
I added something in the READNE about this.